package.json

```diff
@@ -112,7 +112,7 @@
     "@uniswap/sdk": "^3.0.3",
     "@unstoppabledomains/resolution": "^7.1.4",
     "@visx/axis": "^2.10.0",
-    "@visx/brush": "^2.10.1",
+    "@visx/brush": "^2.12.2",
```

Comment:
I upgraded this hoping that it fixed the `findDOMNode` issue (see `patches/PATCHES.md`) but it didn't. Left it because why not be on the latest version?